plugins/turing/runner/experiment_runner.go

```diff
@@ -148,8 +193,11 @@ func NewExperimentRunner(jsonCfg json.RawMessage) (runner.ExperimentRunner, erro
 	r := &experimentRunner{
 		httpClient: client,
 		projectID:  config.ProjectID,
-		passkey:    config.Passkey,
 		parameters: config.RequestParameters,
+		appContext: appCtx,
 	}
+
+	errChannel := make(chan error, 1)
```

Comment:
Can we leave a //TODO note here to handle the error later from `errChannel` later on?